resources/examples/StructuralMechanics.json.json

```diff
@@ -0,0 +1 @@
:21,"mode":0,"inputs":[{"name":"model_import_settings","type":"map","link":259},{"name":"material_import_settings","type":"map","link":260}],"outputs":[{"name":"solver_settings","type":"map","links":[311]}],"properties":{"solver_type":"static","model_part_name":"Structure","domain_size":2,"echo_level":0,"analysis_type":"non_linear","model_import_settings":{"input_type":"mdpa","input_filename":"Ring"},"material_import_settings":{"materials_filename":"StructuralMaterials.json"},"time_stepping":{"time_step":1},"line_search":false,"convergence_criterion":"residual_criterion","displacement_relative_tolerance":0.0001,"displacement_absolute_tolerance":1e-9,"residual_relative_tolerance":0.0001,"residual_absolute_tolerance":1e-9,"max_iteration":10,"rotation_dofs":false,"volumetric_strain_dofs":false}},{"id":206,"type":"parameters/StructuralProjectParameters","pos":[1865,-34],"size":{"0":362.79998779296875,"1":146},"flags":{},"order":22,"mode":0,"inputs":[{"name":"problem_data","type":"map","link":310},{"name":"solver_settings","type":"map","link":311},{"name":"constraints_process_list","type":"process_array","link":308},{"name":"loads_process_list","type":"process_array","link":307},{"name":"list_other_processes","type":"process_array","link":306},{"name":"gid_output","type":"process_array","link":305},{"name":"vtk_output","type":"process_array","link":304}],"outputs":[{"name":"project_parameters","type":"map","links":[312,313]}],"properties":{}},{"id":204,"type":"processes/list","pos":[1523,180],"size":{"0":140,"1":46},"flags":{},"order":9,"mode":0,"inputs":[{"name":"In0","type":"process","link":null}],"outputs":[{"name":"array","type":"process_array","links":[306]},{"name":"size","type":"number","links":null}],"properties":{}},{"id":189,"type":"processes/Vector","pos":[9,513],"size":{"0":210,"1":106},"flags":{},"order":8,"mode":0,"outputs":[{"name":"","type":"process_array","links":[285]}],"properties":{}},{"id":32,"type":"basic/DownloadProblem","pos":[2334,-24],"size":{"0":210,"1":78},"flags":{"collapsed":false},"order":23,"mode":0,"inputs":[{"name":"data","type":0,"link":312},{"name":"download","type":-1,"link":null}],"properties":{"filename":"data.json"}}],"links":[[249,14,0,176,0,"material_array"],[250,176,0,177,0,"string"],[259,172,0,182,0,"map"],[260,177,0,182,1,"map"],[279,197,0,198,1,"process_array"],[280,195,0,198,2,"process_array"],[281,186,0,198,3,"process_array"],[282,149,1,198,0,"string"],[284,199,0,200,1,"process_array"],[285,189,0,200,2,"process_array"],[286,200,0,168,0,"process"],[287,149,2,200,0,"string"],[288,198,0,175,0,"process"],[289,149,0,201,0,"string"],[290,201,0,14,0,"material"],[291,202,0,19,0,"process"],[292,203,0,23,0,"process"],[304,23,0,206,6,"process_array"],[305,19,0,206,5,"process_array"],[306,204,0,206,4,"process_array"],[307,168,0,206,3,"process_array"],[308,175,0,206,2,"process_array"],[310,4,0,206,0,"map"],[311,182,0,206,1,"map"],[312,206,0,32,0,0],[313,206,0,45,0,0]],"groups":[],"config":{},"extra":{},"version":0.4}
```

Comment:
What is this file?

Reply:
That's an example. @WeiqSun97 please change the name so it only contains the extension ´.json´ once

Reply:
OK.